src/main/kotlin/com/sourcegraph/cody/agent/protocol/ProtocolTextDocument.kt

```diff
@@ -106,8 +116,9 @@ private constructor(
       return ProtocolTextDocument(
           uri = uriFor(file),
           content = text,
-          selection = getSelection(editor),
-          visibleRange = getVisibleRange(editor))
+          selection = getSelection(editor, true),
```

Comment:
Ditto here about naming the parameter.

You can also use an enum instead of a boolean to make the meaning of the value obvious.